invenio_app_rdm/records_ui/views/records.py

```diff
@@ -126,6 +126,7 @@ def record_detail(
     pid_value, record, files, media_files, is_preview=False, include_deleted=False
 ):
     """Record detail page (aka landing page)."""
+    record_owner = current_rdm_records.records_service.get_owner(pid_value)
```

Comment:
Comment for reviewers:This returns the User model object which has no dict representation. As I commented [here](https://github.com/inveniosoftware/invenio-rdm-records/pull/1519/files#diff-d79cbaf668935422b2743d81ca9906d9b72ac8b3456c2280ba5a5edacb72796bR157), maybe we should consider creating an interface for the owner as it could potentially be something different than a user.

Reply:
record is already resolved here, can't we resolve the owner directly?